Projects/Contracts/Sources/Journeys/ContractInformation.swift

```diff
@@ -86,6 +84,7 @@ struct ContractInformationView: View {
 
     @ViewBuilder
     private func addCoInsuredView(contract: Contract) -> some View {
+        let _ = store.coInsuredViewModel.initializeCoInsured(with: .init(contract: contract))
```

Comment:
@sladan-hedvig I know this is not the optimal way to do it but not sure how do it differently